src/app/filecoin-plus/allocators/schemas/AllocatorSchema.ts

```diff
@@ -45,9 +45,11 @@ export const AllocatorFileListMetaDataBaseSchema = z.array(
   AllocatorFileMetaDataBaseSchema,
 )
 
-export const AllocatorWithInternalDatacapSchema = AllocatorSchema.extend(
-  InternalDatacapSchema.shape,
-)
+export const AllocatorWithInternalDatacapSchema = AllocatorSchema.extend({
+  address: InternalDatacapSchema.shape.address,
+  remainingDatacap: InternalDatacapSchema.shape.allowance,
+  usedDatacap: InternalDatacapSchema.shape.remainingDatacap,
+})
```

Comment:
Isn't this still weird? 

`remainingDatacap` is `allowance` and `usedDatacap` is `remainingDatacap`?

Reply:
It's confusing, but yes. Here's the screenshot of comparison between Filecoin Plus table and Filecoin Allocators. 

<img width="721" alt="Screenshot 2024-11-13 at 08 38 07" src="https://github.com/user-attachments/assets/0ffea38b-c52b-4fc5-a6d5-216d71a7ee11">
<img width="721" alt="Screenshot 2024-11-13 at 08 38 16" src="https://github.com/user-attachments/assets/7b86490d-4994-4d3a-b940-bfe25aa088cc">

Reply:
We renamed the `remainingDatacap` to `availableDatacap` to avoid confusion.export const 

```
AllocatorWithInternalDatacapSchema = AllocatorSchema.extend({
  address: InternalDatacapSchema.shape.address,
  availableDatacap: InternalDatacapSchema.shape.allowance,
  usedDatacap: InternalDatacapSchema.shape.remainingDatacap, // remainingDatacap shows how much datacap allocator has used
})
```